examples/ts-lb-simple/index.ts

```diff
@@ -23,6 +28,14 @@ const service = new awsx.ecs.FargateService("my-service", {
                     targetGroup: lb.defaultTargetGroup,
                 },
             ],
+            logConfiguration: {
+                logDriver: "awslogs",
+                options: {
+                    "awslogs-group": "my-log-group",
+                    "awslogs-region": "us-west-2",
+                    "awslogs-stream-prefix": "my-app",
+                },
+            },
```

Comment:
Do we need these changes? cc: @thomas11

Reply:
Without them, the container wouldn't start for me when running the tests locally, with error

```
There was an error while retrieving logs from log stream: nginx/nginx/8c06c430fc784875b8f5c28c0e9b5101 in log group: my-service-d975f63.
The specified log stream does not exist.
```

I didn't fully chase it down. Since it seems to work in CI, another region/account difference?